javascript/packages/orchestrator/src/orchestrator.ts

```diff
@@ -375,7 +376,8 @@ export async function start(
 
     // ensure chain raw is ok
     try {
-      const chainRawContent = require(chainSpecFullPath);
+      const chainsSpecString = fs.readFileSync(chainSpecFullPath).toString();
+      const chainRawContent = JSON.parse(chainsSpecString);
```

Comment:
```suggestion
```suggestion
     const chainSpecContent = readAndParseChainSpec(chainSpecFullPathPlain);
```
```